naga/xtask/src/validate.rs

```diff
@@ -13,10 +13,6 @@ use crate::{
     process::{which, EasyCommand},
 };
 
-fn ack_visiting(path: &Path) {
-    log::info!("Validating {}", path.display());
-}
-
```

Comment:
Since we do have an indication of progress, and we do provide full context for jobs that fail, I think it's better to be silent when all its going well. I am not a fan of programs that fill my scrollback with "all is well".

Reply:
> Since we do have an indication of progress

Admittedly, I'd forgotten to consider your usage of `indicatif` here. So, for local usage with a human, LGTM!

I think having `log` output would be preferable for CI, since AFAIK there's not going to be an indication of progress there. I definitely don't want overly verbose output, though, so perhaps we can replace the default `Info`-level `env_logger` initialization with `env_logger::init()`, which would make this `INFO`-level log silent by default instead?

I'll let you decide whether that's something to handle right now, tho. For the purposes of review, I consider this conversation resolved.

Reply:
Dropping the default logging level and then bumping it up for CI sounds like a good idea. But I think it could be handled in a separate PR.

Reply:
Filed as #4938.